test_project/dp_settings.py

```diff
@@ -0,0 +1,35 @@
+from .settings import *
+
+# ENVIRONMENT CONFIG
+SHORT_NAME = 'Djangoplicity'
+TMP_DIR = os.path.join(BASE_DIR, 'tmp')
+
+# DP APPS
+INSTALLED_APPS += [
+    'django.contrib.sites',
+    'djangoplicity.menus',
+    'djangoplicity.pages',
+    'djangoplicity.metadata',
+]
+
+# JAVASCRIPT CUSTOM CONFIG
+JQUERY_JS = "jquery/jquery-1.11.1.min.js"
+JQUERY_UI_JS = "jquery-ui-1.12.1/jquery-ui.min.js"
+JQUERY_UI_CSS = "jquery-ui-1.12.1/jquery-ui.min.css"
```

Comment:
Is it enough to just declare them and not adding the files?

Reply:
Yes!